cola/models/main.py

```diff
@@ -467,6 +467,8 @@ def stage_paths(self, paths):
 
         for path in set(paths):
             if core.exists(path):
+                if path[-1:] == '/':
+                    path = path[:-1]
```

Comment:
A slightly more idiomatic python way to write this might be,

```python

if path.endswith('/'):
    path = path.rstrip('/')
```

They're both just about equivalent, except the `rstrip()` will consume multiple `/`, but that doesn't matter since we never see those in practice.